### PR TITLE
Fix #75 Work can not have have null work type

### LINKED ISSUE #75: Works should have a non-null work type
https://github.com/DakaraProject/dakara-server/blob/b7b11cc0fd9605a4312d6ed36b20f97954b8d5b0/dakara_server/library/models.py#L46

According to the code, a work can have a null work type, but this does not make any sense. I think we can remove this behavior.

### PR DESCRIPTION
**Pull request by Nadeflore**
Update feeder to ignore work when no worktype is provided
Update tests to match this new behaviour

### CODE REVIEW

File: dakara_server/library/tests_command_feed.py [hunk 1]
```diff
@@ -739,3 +739,36 @@ def test_feed_command_artist_work_tag_change(self):
...
+
+            songs = Song.objects.all()
+            self.assertEqual(len(songs), 1)
+            song = songs[0]
+            self.assertEqual(song.filename, filename)
+            # Check artist
+            self.assertEqual(len(song.artists.all()), 1)
+            self.assertEqual(song.artists.all()[0].name, "Artist name")
+            # Check work was not created
+            self.assertEqual(len(song.works.all()), 0)
```
[the remark this answers is not in the document]
Comment (Nadeflore) [reply dakara_server/library/tests_command_feed.py]: Sorry, I'm not sure I understand what you mean,
there does not appear to be a `length` method on a `ManyRelatedManager`

[the remark this answers is not in the document]
Comment (Neraste) [reply dakara_server/library/tests_command_feed.py]: Sorry, I meant `count`.

[the remark this answers is not in the document]
Comment (Nadeflore) [reply dakara_server/library/tests_command_feed.py]: I guess you meant `count()` juste like above_

[the remark this answers is not in the document]
Comment (Neraste) [reply dakara_server/library/tests_command_feed.py]: Damn JS…

File: dakara_server/library/tests_command_feed.py [hunk 2]
```diff
@@ -739,3 +739,36 @@ def test_feed_command_artist_work_tag_change(self):
...
             self.assertEqual(song_new.tags.all()[0].name, "NEWTAG")
+
+    def test_feed_command_no_work_type(self):
+        """Test feed command when parser return work but not worktype
+           Work should not be created, because a work can not exists
+           without worktype
+        """
+        # Pre-Assertions
+        songs = Song.objects.all()
+        self.assertEqual(len(songs), 0)
```
[the remark this answers is not in the document]
Comment (Nadeflore) [reply dakara_server/library/tests_command_feed.py]: actually `Song.objects.count()` works as well.

**Comment by Nadeflore** [comment]
@Neraste I made the requested changes.
I'm waiting for your approval on this PR

File: dakara_server/library/tests_command_feed.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Neraste) [reply dakara_server/library/tests_command_feed.py]: This is better

**Pull request merged**